services/submission.js

```diff
@@ -180,6 +181,30 @@ class Submission {
         }, {"$limit": 1}]);
         return (result?.length > 0) ? result[0] : null;
     }
+
+    async getSubmission(params, context){
+        verifySession(context)
+            .verifyInitialized()
+            .verifyRole([ROLES.SUBMITTER, ROLES.ORG_OWNER, ROLES.DC_POC, ROLES.FEDERAL_LEAD, ROLES.CURATOR, ROLES.ADMIN]);
+        const id = params?._id
+        const rUser = await this.userService.getUserByID(context?.userInfo?._id);
+        const aSubmission = await this.findByID(id);
+        if(!aSubmission){
+            throw new Error(ERROR.INVALID_SUBMISSION_NOT_FOUND)
+        }else{
+            // view condition
+            const Condition_DC_POC  = (context?.userInfo?.role === ROLES.DC_POC )&& (rUser?.dataCommons.includes(aSubmission?.dataCommons))
```

Comment:
Please, use a camelcase. And remove a unnecessary space after Condition_DC_POC. Apply entire variables.